tgui/packages/tgui/interfaces/Autolathe.tsx

```diff
@@ -0,0 +1,254 @@
+import { useBackend } from '../backend';
+import { LabeledList, Section, ProgressBar, Collapsible, Stack, Icon, Box, Tooltip, Button } from '../components';
+import { Window } from '../layouts';
+import { capitalize } from 'common/string';
+import { Design, MaterialMap } from './Fabrication/Types';
+import { DesignBrowser } from './Fabrication/DesignBrowser';
+import { classes } from 'common/react';
+import { MaterialCostSequence } from './Fabrication/MaterialCostSequence';
+import { Material } from './Fabrication/Types';
+
+type AutolatheDesign = Design & {
+  buildable: boolean;
```

Comment:
Byond is unable to send booleans so technically this type is wrong, you can use a `BooleanLike` alias or `number`.